dbms/src/DataStreams/MySQLBlockOutputStream.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "IBlockOutputStream.h"
+#include <Core/MySQLProtocol.h>
+#include <Formats/FormatFactory.h>
+#include <Formats/FormatSettings.h>
+#include <Interpreters/Context.h>
+
+namespace DB
+{
+
+/** Interface for writing rows in MySQL Client/Server Protocol format.
+  */
+class MySQLBlockOutputStream : public IBlockOutputStream
```

Comment:
It should be named `MySQLWireBlockOutputStream` to avoid confusion with `MySQLBlockInputStream`.